db-service/test/cqn4sql/DELETE.test.js

```diff
@@ -69,80 +69,101 @@ describe('DELETE', () => {
+            "as": "author2"
+          }
+        }
+      }`)
+    expected.DELETE.where = [
+      {
+        list: [
+          {
+            ref: ['author2', 'ID'],
+          },
+        ],
+      },
+      'in',
+      subquery,
+    ]
+    expect(query.DELETE).to.deep.equal(expected.DELETE)
+  })
 
   it('DELETE with assoc filter and where exists expansion', () => {
     const { DELETE } = cds.ql
     let d = DELETE.from('bookshop.Reproduce[author = null and ID = 99]:accessGroup')
     const query = cqn4sql(d)
 
     const expected = {
-      "DELETE": {
-        "from": {
-          "ref": [
-            "bookshop.AccessGroups"
-          ],
-          "as": "accessGroup"
+      DELETE: {
+        from: {
+          ref: ['bookshop.AccessGroups'],
+          as: 'accessGroup',
         },
-        "where": [
-          "exists",
+        where: [
+          'exists',
           {
-            "SELECT": {
-              "from": {
-                "ref": [
-                  "bookshop.Reproduce"
-                ],
-                "as": "Reproduce"
+            SELECT: {
+              from: {
+                ref: ['bookshop.Reproduce'],
+                as: 'Reproduce',
               },
-              "columns": [
+              columns: [
                 {
-                  "val": 1
-                }
+                  val: 1,
+                },
               ],
-              "where": [
+              where: [
                 {
-                  "ref": [
-                    "Reproduce",
-                    "accessGroup_ID"
-                  ]
+                  ref: ['Reproduce', 'accessGroup_ID'],
                 },
-                "=",
+                '=',
                 {
-                  "ref": [
-                    "accessGroup",
-                    "ID"
-                  ]
+                  ref: ['accessGroup', 'ID'],
                 },
-                "and",
+                'and',
                 {
-                  "xpr": [
+                  xpr: [
                     {
-                      "ref": [
-                        "Reproduce",
-                        "author_ID"
-                      ]
+                      ref: ['Reproduce', 'author_ID'],
                     },
-                    "=",
+                    '=',
                     {
-                      "val": null
-                    }
-                  ]
+                      val: null,
+                    },
+                  ],
                 },
-                "and",
+                'and',
                 {
-                  "ref": [
-                    "Reproduce",
-                    "ID"
-                  ]
+                  ref: ['Reproduce', 'ID'],
                 },
-                "=",
+                '=',
                 {
-                  "val": 99
-                }
-              ]
-            }
-          }
-        ]
-      }
+                  val: 99,
+                },
```

Comment:
Why are there so many trailing commas?

Reply:
Because of this:
https://github.com/cap-js/cds-dbs/blob/7896289ec97f28bdc2c1d7700af1953608c96ef2/.prettierrc.js#L16